torchvision/prototype/transforms/_transform.py

```diff
@@ -35,6 +35,9 @@ def forward(self, *inputs: Any) -> Any:
 
         params = self._get_params(flat_inputs)
 
+        # TODO: right now, any tensor or datapoint passed to forward() will be transformed.
+        # The rest is bypassed.
+        # What if there are tensor parameters that users don't want to be transformed? Is that a plausible scenario?
```

Comment:
We previously agreed that we should have a no-op datapoint that will never be transformed. There was some discussion to use the plain `Datapoint` class directly or add another thin wrapper subclass as suggested in https://github.com/pytorch/vision/pull/6663#discussion_r982567437